ansible/README_DISTRIBUTED.md

```diff
@@ -0,0 +1,101 @@
+Deploying Distributed OpenWhisk using Ansible
+=========
+
+**Caveat:** All Ansible commands are meant to be executed from the `ansible` directory.
+This is important because that's where `ansible.cfg` is located which contains generic settings that are needed for the remaining steps.
+
+By default, if you omit the `-i` parameter in ansible commands, the `local` environment will be used.
+
+In all instructions, replace `<openwhisk_home>` with the base directory of your OpenWhisk source tree. e.g. `openwhisk`
+
+Login to your bootstrapper VM. Your local machine can act as the bootstrapping machine as well, if it can connect to the VMs deployed in your IaaS.
+
+This installs modules and packages to be able to manage cloud instances via ansible.
+
+```
+sudo apt-get -y install python-setuptools python-dev libssl-dev
+sudo pip install shade pytz positional appdirs monotonic rfc3986
```

Comment:
These commands should move under "distributed deployment using OpenStack as Iaas". Collapse them with the two below accordingly (one apt-get, one pip install)